var/spack/repos/builtin/packages/libice/package.py

```diff
@@ -18,3 +18,12 @@ class Libice(AutotoolsPackage, XorgPackage):
     depends_on('xtrans')
     depends_on('pkgconfig', type='build')
     depends_on('util-macros', type='build')
+
+    @property
+    def libs(self):
+        for dir in ['lib64', 'lib']:
+            libs = find_libraries('libICE', join_path(self.prefix, dir),
+                                  shared=True, recursive=False)
+            if libs:
+                return libs
+        return None
```

Comment:
```suggestion
        return find_libraries('libICE', self.prefix, shared=True, recursive=True)
```
Same for other packages.

Reply:
done, thanks.